Add Data Privacy Page and Improve responsiveness of pages

Seacrs opened the pull request:
This PR adds the missing shapes to the hero section and implements responsiveness across both the index and imprint pages. It also introduces the data privacy page that was previously missing, ensuring full consistency with the Figma design.

🔗 **Notes**
To access the new pages:

- Imprint page: via the Services link
- Data privacy page: via the Contacts link

mucyoelie committed:
add privacy policy

mucyoelie committed:
add header or Navbar

mucyoelie committed:
update privacy page

mucyoelie committed:
update privacy page

mucyoelie committed:
update privacy page

elyse502 committed:
feat(imprint): improve layout and responsiveness of imprint section

elyse502 committed:
feat(imprint-page): update imprint and footer sections with redesigned structure and layout

    - Removed redundant red polygon section from 'imprint.html' and adjusted main content loading.

    - Refactored 'footer.html' with a new responsive structure, including red polygon graphic, improved contact information, and updated navigation links.

    - Enhanced overall layout and semantic structure to improve readability, maintainability, and visual hierarchy.

mucyoelie committed:
fix Merge confilct

mucyoelie committed:
fix Merge confilct

mucyoelie committed:
update privacy page

Eric-mar committed:
adding output css

elyse502 committed:
feat: resolve conflicts

elyse502 committed:
feat: resolve conflicts

elyse502 committed:
feat: resolve conflicts

elyse502 committed:
refactor: move imprint content from hero.html to info.html and update imprint page structure

Eric-mar committed:
feat: add shapes to hero section

Eric-mar committed:
Fix Merge conflict

Eric-mar committed:
Fix: add main section

Seacrs committed:
Feat: optimize view for all screen sizes

mucyoelie committed:
update privacy page

mucyoelie committed:
update privacy page

elyse502 committed:
fix(responsive): resolve layout overflow issue on tablets in info section

    - Removed unnecessary Tailwind class that caused overflow on mid-sized screens like iPad Pro.

Seacrs committed:
fix: add sticky function on main section polygon

Seacrs committed:
chore: adjust content for screen sizes

Seacrs committed:
fix issue

Seacrs committed:
style: add shape to header section for mobile version

Seacrs committed:
Merge pull request #9 from mucyoelie/shapes

This PR adds polygons to the hero section of the page

Seacrs committed:
style: improve visibility of heade

Seacrs committed:
merge dev

elyse502 committed:
merge: resolve conflicts between dev and ft/imprint-page branches

Seacrs committed:
style: adjust main and header sections for responsive screens

Seacrs committed:
style: adjust main section content for responsive screens

Seacrs committed:
Merge dev to ft/index.html

mucyoelie committed:
change stylesheet location

mucyoelie committed:
Merge dev to privacy

Seacrs committed:
Merge branch 'dev' into privacy

Seacrs committed:
Merge pull request #14 from mucyoelie/privacy

Completed Desktop Version of Privacy policy page

Seacrs committed:
Merge branch dev to ft/index.html

mucyoelie committed:
Merge pull request #13 from mucyoelie/ft/index.html

Main and Navbar responsiveness

mucyoelie committed:
update index.html

Seacrs committed:
Merge branch dev to ft/index.html

mucyoelie committed:
Merge branch 'dev' into privacy

Eric-mar committed:
solving issues from the index.html branch

Seacrs committed:
Style: add polygon responsiveness on hero section

Seacrs committed:
Pull change from origin

elyse502 committed:
feat(hero): implement responsive hero section with polygon designs

    - Add responsive polygon shapes with proper z-index layering

    - Implement mobile-first responsive design patterns

    - Enable hero section loading in main layout

    - Update header z-index for proper stacking context

elyse502 committed:
feat(header): make logo clickable and improve navigation accessibility

elyse502 committed:
feat(navigation): implement internal page navigation with anchor links

elyse502 committed:
refactor(hero): optimize hero text styling and structure

elyse502 committed:
feat(hero): enhance user experience with animated polygons

Seacrs committed:
feat: add close button to phone and Ipad screens

Seacrs committed:
fix: make hero and footer visible

Fuad-Jamal commented on src/sections/footer.html:
```diff
@@ -1,40 +1,52 @@
+      viewBox="0 0 722 418"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      class="sticky top-0 h-[190px] w-[330px] md:h-[244px] md:w-[470px] lg:h-[420px] lg:w-[722px] z-10"
```
You have to refrain from using arbitrary values

Fuad-Jamal commented on src/sections/header.html:
```diff
@@ -1,20 +1,58 @@
+  >
     <div class="place-self-start justify-self-start">
-      <p class="text-[20px] font-normal">DE</p>
-      <p class="text-[20px] font-normal">EN</p>
```
This has to be a link

elyse502 committed:
chore(deps): update Tailwind CSS configuration and tracking

elyse502 committed:
chore(deps): revert Tailwind CSS CLI and update dependency versions

elyse502 committed:
chore(build): track dist directory and update gitignore

elyse502 committed:
refactor: improve page metadata and content structure

elyse502 committed:
fix(navigation): restore aboutus ID for anchor navigation

Seacrs committed:
Merge branch Dev to ft/index.html

Seacrs committed:
Merge Dev to ft/index.html

Seacrs committed:
Merge Dev to ft/index.html

Seacrs committed:
feat: Add mobile menu

elyse502 committed:
feat(hero): redesign layout with improved polygon positioning and responsive scaling

Seacrs committed:
chore: change links to other pages

elyse502 committed:
refactor(hero): optimize layout structure and improve responsive positioning

Seacrs committed:
chore: add links to page sections

elyse502 committed:
fix(hero): adjust container height and text positioning

Seacrs committed:
style: Add polygon shapes to phone and tablet menu

Eric-mar committed:
adding shapes react

Seacrs committed:
fix: remove unwanted file in branch

elyse502 committed:
refactor(ui): replace arbitrary pixel values with standard Tailwind classes for responsive consistency

elyse502 committed:
refactor(ui): replace arbitrary pixel values with standard Tailwind classes for responsive consistency

Seacrs committed:
feat: Add navbar to mobile screens and adjust red polygon for visibility

Seacrs committed:
Merge origin/dev to dev

Seacrs committed:
Merge branch dev to branch ft/index.html

Seacrs committed:
Style: add missing tailwind styles

Seacrs committed:
chore: Move script.js from src to scripts folder

elyse502 committed:
Merge branch 'ft/hero-section' into dev, resolved conflict in dist/output.css

Seacrs committed:
Merge origin/dev into dev

Seacrs committed:
Merge branch dev into branch ft/index.html

mucyoelie committed:
make responsiveness to privacy page

mucyoelie reviewed (approved):
Nice job maintaining consistency with the existing codebase

Seacrs committed:
Made changes

Seacrs committed:
Merge origin/dev to dev

Seacrs committed:
Merge origin/dev to dev

Seacrs committed:
Merge origin/dev to dev

Seacrs committed:
Merge dev to ft/index.html

elyse502 committed:
merge: resolve merge conflicts and sync hero updates

elyse502 committed:
merge: integrate privacy feature branch and resolve conflicts

Seacrs committed:
style: Add responsive menu on Iphone 14

Seacrs committed:
Merge origin/dev to dev

Seacrs committed:
Merge branch dev to ft/index.html

elyse502 committed:
style: enhance spacing and layout across multiple sections

Seacrs committed:
style: increase big green polygon width  on mobile screen

elyse502 committed:
style: fine-tune section spacing for optimal visual balance

Seacrs committed:
Merge dev into ft/index.html

Seacrs committed:
Merge origin/dev into dev

Seacrs committed:
Merge branch dev into branch ft/index.html

elyse502 committed:
docs: enhance project documentation and README

Eric-mar committed:
changing the footer

Eric-mar reviewed (approved):
Good job 👍

Seacrs committed:
Merge branch 'dev' into ft/index.html

elyse502 committed:
merge: resolve conflicts and integrate header updates

elyse502 committed:
Merged 'docs/readme' into 'dev' to update README documentation

Seacrs committed:
Merge pull request #19 from mucyoelie/ft/index.html

Menu bar responsiveness

elyse502 committed:
feat: add green header to legal pages and optimize loading

    - Implement greenHeader.html for imprint and privacy pages

    - Replace standard header with green variant on legal pages

    - Disable page.js script to prevent functionality conflicts

    - Maintain header.js for core navigation functionality

elyse502 committed:
fix: correct green header styling and mobile menu visibility
